test/studies/HDFS/tzakian/HDFS.chpl

```diff
@@ -54,3 +58,280 @@ extern proc hdfsOpenFile(hdfsFS: c_void_ptr, path: c_string, flags: c_int, buffe
 /* Returns tSize (int32) */
 extern proc hdfsWrite(hdfsFS: c_void_ptr, hdfsFile: c_void_ptr, buffer: c_string, length: int(32)): int(32);
 
+
+/* ====================== TODO's ========================
```

Comment:
Where is this content coming from, and should it be removed elsewhere?

Reply:
This content is from `test/studies/HDFS/tzakian/HDFSiterator.chpl`.  I will remove this file.